app/models/user.rb

```diff
@@ -0,0 +1,19 @@
+class User < ActiveRecord::Base
+	has_many :ideas, foreign_key: :creator_id
+	has_many :comments, foreign_key: :author_id
+	has_many :votes, foreign_key: :voter_id
```

Comment:
@stephmarx @rfleury2 We discussed a bit about what we want to live on if a user wants to remove themselves, that the idea will still live on, given that commenters will want to still engage with the idea.. do you feel that we should add dependent destroy on anything?